src/components/application_manager/src/commands/mobile/set_app_icon_request.cc

```diff
@@ -151,9 +150,16 @@ void SetAppIconRequest::CopyToIconStorage(
       return;
     }
 
-    while (!IsEnoughSpaceForIcon(file_size)) {
+    while (!IsEnoughSpaceForIcon(file_size) && is_icons_saving_enabled_) {
       RemoveOldestIcons(icon_storage, icons_amount);
     }
+    if (!is_icons_saving_enabled_ &&
+        (storage_max_size < (file_size + storage_size))) {
+      LOG4CXX_WARN(logger_,
+                   "Unable to got enough space for storing icon. "
```

Comment:
@VVeremjova Missed this one, I think it should be "to get" instead of "to got" here.